modules/smtp/site.js

```diff
@@ -512,14 +512,14 @@ $(function () {
 
             // if contact_cc not exist in contact list for user
             var checkInList = "";
-            if (list_emails) {
+            if (typeof list_emails !== "undefined") {
```

Comment:
Migrate this fonctionnality into contact module.

Reply:
it is related to the contact module, by leaving it here you create a dependency of the smtp module to the contact module

Reply:
Done